### PR TITLE
Filtered platform inventory kafka messages via its headers, modified …

### PR DESCRIPTION
**Pull request by AaronH88**
…kafka message logging to only log on created events.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `go fmt ./...` to check that my code is properly formatted
- [X] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: pkg/services/consumers.go [hunk 1]
```diff
@@ -134,31 +133,28 @@ func (s *KafkaConsumerService) ConsumeInventoryCreateEvents() error {
...
-			if e.Type == "created" {
-				err = s.DeviceService.ProcessPlatformInventoryCreateEvent(m.Value)
-				if err != nil {
-					log.WithFields(log.Fields{
-						"error": err,
-					}).Error("Error writing Kafka message to DB")
-				}
-			} else {
-				log.Debug("Skipping message none create message from platform insights")
+		var service string
```
Comment (biancarosa) [inline pkg/services/consumers.go:136]:
```suggestion
		var event_type string
```

maybe?

Comment (AaronH88) [reply pkg/services/consumers.go:136]: Good Idea, Ill make the change

**Review by biancarosa — APPROVED** [review]
lgtm

File: pkg/services/consumers.go [hunk 2]
```diff
@@ -134,30 +133,27 @@ func (s *KafkaConsumerService) ConsumeInventoryCreateEvents() error {
...
+		}
+		if eventType == "created" {
+			err = s.DeviceService.ProcessPlatformInventoryCreateEvent(m.Value)
+			if err != nil {
+				log.WithFields(log.Fields{
+					"error": err,
+				}).Error("Error writing Kafka message to DB")
 			}
+		} else {
+			log.Debug("Skipping message none create message from platform insights")
```
Comment (loadtheaccumulator) [inline pkg/services/consumers.go:156]: Same text change as in devices.go.

/none/- not a/

**Review by loadtheaccumulator — COMMENTED** [review]
lgtm w/o the one text change to match the other.

File: pkg/services/consumers.go (hunk 2 above)
Comment (AaronH88) [reply pkg/services/consumers.go]: Changed, thanks for spotting this!

**Pull request merged**